bugwarrior/docs/services/github.rst

```diff
@@ -135,6 +135,21 @@ issues matched by the query, set::
     github.include_user_issues = False
     github.include_user_repos = False
 
+Include Certain Issues
+++++++++++++++++++++++
+
+Since Github doesn't allow to show on the website or even directly with their
+API issues a user is subscribed to (see [#] and [#]), Bugwarrior allows to
+specify specific issues you'd like to be added to your Taskwarrior database::
+
+    github.include_these_issues = octokit/octokit.rb/issues/1057, octokit/octokit.rb/issues/957
+
+Note that you need to specify only the part that comes after the host and the
+scheme, usually what's after ``https://github.com/``, including the last ``/``.
+
+.. [#] https://github.com/dear-github/dear-github/issues/247
```

Comment:
Maybe update these links to show the new subscription UI, but mention the lack of an API - which may help people understand what this is for.